settings.gradle.kts

```diff
@@ -0,0 +1,59 @@
+rootProject.name = "hex-thai-star"
+
+include(":booking-api-out")
+include(":booking-adapter-out-jdbi")
+include(":order-application")
+include(":booking-adapter-out-quarkusmail")
+include(":common-domain")
+include(":order-domain")
+include(":booking-adapter-in-springweb")
+include(":order-adapter-out-jpa")
+include(":awslambda")
+include(":common-adapter-out-jdbi")
+include(":booking-adapter-in-quarkusweb")
+include(":booking-adapter-out-springmail")
+include(":springboot")
+include(":booking-adapter-out-awsmail")
+include(":booking-application")
+include(":order-adapter-in-springweb")
+include(":order-api-out")
+include(":quarkus")
+include(":order-api-in")
+include(":order-adapter-out-booking")
+include(":quarkus-awslambda")
+include(":booking-adapter-out-dynamodb")
+include(":booking-adapter-in-awslambda")
+include(":booking-adapter-out-jpa")
+include(":archunit")
+include(":common-application")
+include(":booking-domain")
+include(":booking-api-in")
+
+project(":booking-api-out").projectDir = file("booking/booking-api-out")
```

Comment:
po co te rzeczy? do czego jest to potrzebne?

Reply:
nie znalazlem bezposredniej informacji w necie, ale sadze, ze to potrzeba, bo mamy sciezke katalogow

Reply:
znalazlem w doku i poprawilem, tak jak myslalem chodzi o sciezke katalogu

     * <p>Adds the given projects to the build. Each path in the supplied list is treated as the path of a project to
     * add to the build. Note that these path are not file paths, but instead specify the location of the new project in
     * the project hierarchy. As such, the supplied paths must use the ':' character as separator (and NOT '/').</p>
     *
     * <p>The last element of the supplied path is used as the project name. The supplied path is converted to a project
     * directory relative to the root project directory. The project directory can be altered by changing the 'projectDir'
     * property after the project has been included (see {@link ProjectDescriptor#setProjectDir(File)})</p>
     *
     * <p>As an example, the path {@code a:b} adds a project with path {@code :a:b}, name {@code b} and project
     * directory {@code $rootDir/a/b}. It also adds the a project with path {@code :a}, name {@code a} and project
     * directory {@code $rootDir/a}, if it does not exist already.</p>